src/node.rs

```diff
@@ -583,8 +584,13 @@ impl PbftNode {
     /// Handle a `BlockNew` update from the Validator
     ///
     /// The validator has received a new block; check if it is a block that should be considered,
-    /// add it to the log as an unvalidated block, and instruct the validator to validate it.
-    pub fn on_block_new(&mut self, block: Block, state: &mut PbftState) -> Result<(), PbftError> {
+    /// add it to the log as an unvalidated block   , and instruct the validator to validate it.
```

Comment:
It looks like this comment was unintentionally altered; please remove this change.

Reply:
Noted . Will do the changes .Thanks.